src/java.base/unix/native/libnio/ch/FileChannelImpl.c

```diff
@@ -50,13 +50,10 @@
 #include "java_lang_Integer.h"
 #include <assert.h>
 
-static jfieldID chan_fd;        /* jobject 'fd' in sun.nio.ch.FileChannelImpl */
-
 JNIEXPORT jlong JNICALL
 Java_sun_nio_ch_FileChannelImpl_initIDs(JNIEnv *env, jclass clazz)
```

Comment:
Hello Brian, now that this function doesn't initialize ids, should it instead be renamed to something else? Also the corresponding `initIDs` method on the Java side? And perhaps remove this comment from `FileChannelImpl.java`:

```
    // Caches fieldIDs
    private static native long initIDs();
```

Reply:
Yeah I had thought of that as well but shined it on. I'll update before integrating.